sdk/search/test-resources.json

```diff
@@ -1,8 +1,121 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
-    "contentVersion": "1.0.0.0",
-    "parameters": {},
-    "variables": {},
-    "resources": [],
-    "outputs": {}
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "baseName": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().name]",
+      "metadata": {
+        "description": "The base resource name."
+      }
+    },
+    "searchEndpointSuffix": {
+      "type": "string",
+      "defaultValue": "search.windows.net",
+      "metadata": {
+        "description": "The Search service endpoint suffix."
+      }
+    },
+    "storageEndpointSuffix": {
+      "type": "string",
+      "defaultValue": "search.windows.net",
```

Comment:
storage end point 🤔

Reply:
This still doesn't resolve the issue of making the storage endpoint value itself to changed/configurable when the tests are running in any other Azure cloud. But for now, we can add a separate issue to track this. 

cc: @danieljurek

Reply:
This is what I get for copy and pasting.

Reply:
Let's file an issue for this as this affects more than just Search.

Reply:
This looks OK to me. A default value here is fine and I can pass overrides in at the matrix-level when we're testing against other clouds.